added image alt attribute to canvas_grid_translate.png (#22228)
mdn/content: files/en-us/web/api/canvasrenderingcontext2d/translate/index.md

matrix by moving the canvas and its origin `x` units horizontally and
`y` units vertically on the grid.

> ![Grid with cross illustrates how the canvas and origin move based off the values of x and y.](canvas_grid_translate.png)

Reviewer: ```suggestion
![A canvas's origin moved on the x and y axes based on the values of the translate method.](canvas_grid_translate.png)
```
